docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -44,6 +44,14 @@ User-defined dependencies will be appended to the generated unit file, but any e
 
 Set the systemd unit name prefix for containers. The default is *container*.
 
+#### **--env**, **-e**=*env*
+
+Set environment variables to the systemd unit files.
+
+This option allows arbitrary environment variables that are available for the process to be launched inside of the container. If an environment variable is specified without a value, Podman will check the host environment for a value and set the variable only if it is set on the host. As a special case, if an environment variable ending in __*__ is specified without a value, Podman will search the host environment for variables starting with the prefix and will add those variables to the container.
```

Comment:
I don't see a test checking whether the variables are really injected _into_ the container.  The changes don't suggest that it works. Looking at the original issue, it should be enough to make the variables available to Podman but they are not required _inside_ the container.

Reply:
Thank you for reviewing.
Certainly, this command is only add environment variables to the systemd unit files.
This means that not to set the environment variables to the containers directly.
(I valified --env sets the environment variables to the containers correctly by systemd.)

So, I removed the text about "inside the container".